scripts/hypothesistooling.py

```diff
@@ -404,3 +405,23 @@ def should_run_ci_task(task, is_pull_request):
     else:
         print('There are no changes which would need a test run.')
         return False
+
+
+SECRETS_BASE = os.path.join(ROOT, 'secrets')
+SECRETS_TAR = SECRETS_BASE + '.tar'
+ENCRYPTED_SECRETS = SECRETS_TAR + '.enc'
+
+DEPLOY_KEY = os.path.join(ROOT, 'deploy_key')
+PYPIRC = os.path.join(ROOT, '.pypirc')
+
+
+def decrypt_secrets():
+    subprocess.check_call([
+        'openssl', 'aes-256-cbc', '-K', '$encrypted_39cb4cc39a80_key',
```

Comment:
Do you want `os.environ['encrypted_39cb4cc39a80_key']`? Shell variables aren’t going to be expanded here, are they?

Reply:
Oh! Good point. Thank you.